.github/CODEOWNERS

```diff
@@ -1 +1 @@
- * @einride/transportation-platform @thall
+ * @einride/freight-control @thall
```

Comment:
We would have to add the team to the repo settings.

Reply:
Done